app/src/main/res/values/strings.xml

```diff
@@ -842,6 +842,15 @@ Before uploading your changes, the app checks with a &lt;a href=\"https://www.we
     <string name="quest_camera_type_fixed">Fixed</string>
     <string name="quest_camera_type_panning">Panning</string>
 
+    <string name="quest_camp_drinking_water_title">Is drinking water available to all customers here?</string>
+    <string name="quest_camp_shower_title">Does it have showers?</string>
+    <string name="quest_camp_power_supply_title">Can any customer charge their phone/laptop here?</string>
+    <string name="quest_camp_type_title">Who may camp here?</string>
+    <string name="quest_camp_type_tents_and_caravans">Both tents and caravans</string>
+    <string name="quest_camp_type_caravans_only">Only caravans</string>
+    <string name="quest_camp_type_tents_only">Only tents</string>
+    <string name="quest_camp_type_backcountry">Backcountry camping (no facilities)</string>
```

Comment:
Putting it in the main answer section would IMHO create more confusion than it would help.

Also, it's not a big loss if it is not found - it mostly just changes implied defaults / skips the extra quests (see https://github.com/streetcomplete/StreetComplete/issues/4198#issuecomment-1179793163 and below) - initially I wasn't even planning to include it.